packages/esm-billing-app/src/types/index.ts

```diff
@@ -493,3 +488,56 @@ export type PaymentMode = {
   sortOrder?: number | null;
   resourceVersion: string;
 };
+
+export type SHAIntervension = {
+  interventionCode: string;
+  shaCategory: string;
+  accessCode: string;
+  subCategoryBenefitsPackage: string;
+  interventionName: string;
+  needsPreAuth: string;
+  categoryHealthFacility: string;
+  providerPaymentMechanism: string;
+  gender: string;
+  limitIndividualHousehold: string;
+  interventionQuantityYear: string;
+  tariffs: string;
+  limitIndividual: string;
+  minAge: string;
+  maxAge: string;
+  limitsIndividualHousehold: string;
+  phc: string;
+  shif: string;
+  eccif: string;
+  automateManual: string;
+};
+
+export interface _SHAIntervension {
+  id: number;
+  intervention_code: string;
```

Comment:
```suggestion
  intervention_code: string;
```
Should we be using camelCase here